cpu/sam0_common/periph/spi.c

```diff
@@ -117,6 +118,10 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
     /* power on the device */
     poweron(bus);
 
+    /* disable the device */
+    dev(bus)->CTRLA.reg &= ~(SERCOM_SPI_CTRLA_ENABLE);
+    while (dev(bus)->SYNCBUSY.reg & SERCOM_SPI_SYNCBUSY_ENABLE) {}
```

Comment:
Taken the fact of basically moving this from `spi_release` to `spi_aquire` makes me wonder if there is some combination where the SPI device gets configured but is enabled.

>The following registers are enable-protected, meaning that they can only be written when the SPI is disabled (CTRL.ENABLE is zero):
>*  Control A register (CTRLA), except Enable (CTRLA.ENABLE) and Software Reset (CTRLA.SWRST)
>*  Control B register (CTRLB), except Receiver Enable (CTRLB.RXEN)
>*  Baud register (BAUD)
>*  Address register (ADDR)